src/lwc/geBatchWizard/geBatchWizard.js

```diff
@@ -352,6 +359,33 @@ export default class geBatchWizard extends NavigationMixin(LightningElement) {
             dataImportBatch.fields[DATA_IMPORT_BATCH_ID_INFO.fieldApiName] = this.recordId;
         }
 
+        dataImportBatch = this.stripNameSpacedDonationMatchingRuleFields(dataImportBatch);
+
+        return dataImportBatch;
+    }
+
+    /***************************************************************************
+     * @description Strips namespace prefix from the Data Import Batch donation matching
+     * rule fields if in a namespaced context
+     * @param dataImportBatch
+     * @returns {object}
+     */
+    stripNameSpacedDonationMatchingRuleFields (dataImportBatch) {
```

Comment:
One final question. What do you think about renaming this to something like below?
```suggestion
    stripNamespaceFromDonationMatchingRuleFields (dataImportBatch) {
```